test/test.dart

```diff
@@ -191,7 +191,7 @@ Future<void> tryDeleteFile(FileSystemEntity fileEntity, {bool recursive = false}
       await fileEntity.delete(recursive: recursive);
       break;
     } catch (e) {
-      await Future<void>.delayed(Duration(milliseconds: 50));
```

Comment:
This was like that in my PR: `await Future<dynamic>.delayed(Duration(milliseconds: 150));` https://github.com/realm/realm-dart/commit/fe3a5ef9ebe5387705dbdcd0c710616a438e2031
I'm afraid there could be also other changes that were not merged.